File: PRAKTIKUM-PEMROGRAMAN1/MODUL6/Python/PRAK604-2410817210024-MuhammadIbnuSina.py
kode = input().rstrip()
pesan = input().rstrip()

bintang=0
pagar=0

hasil = ""
for i in range(len(kode)):
    if kode[i]==pesan[i]:
        hasil += '*'
        bintang += 1
    else:
        hasil += '#'
        pagar += 1

if len(kode)!=len(pesan):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    print(hasil)
    print(f"* = {bintang}")
    print(f"# = {pagar}")

    if bintang >= pagar:
        print("Pesan Asli")
    else:
        print("Pesan palsu")

File: PRAKTIKUM-PEMROGRAMAN1/MODUL6/C/PRAK604-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>
#include <string.h>

int main() {
    char kode[100], pesan[100];
    int bintang=0, pagar=0;
    scanf("%[^\n]%*c", &kode);
    scanf("%[^\n]%*c", &pesan);


    for (int i=0; i<strlen(kode); i++) {
        if (kode[i]==pesan[i]) {
            pesan[i]='*';
            bintang++;
        } else {
            pesan[i]='#';
            pagar++;
        }
    }

    if (strlen(kode) != strlen(pesan)) {
        printf("Panjang kalimat berbeda, pesan palsu\n");
    } else {
        printf("%s\n", pesan);
        printf("* = %d\n", bintang);
        printf("# = %d\n", pagar);

        if (bintang>=pagar) {
            printf("Pesan Asli\n");
        } else {
            printf("Pesan palsu\n");
        }
    }
    return 0;
}